tests/cypress/integration/issue_1785_propagation_latest_frame.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check propagation work from the latest frame', () => {
+
+    const issueId = '1785'
+    const labelName = `Issue ${issueId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const images = [`image_${issueId}_1.png`,
+                    `image_${issueId}_2.png`,
+                    `image_${issueId}_3.png`]
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+    const archiveName = `images_issue_${issueId}.zip`
+    const archivePath = `cypress/fixtures/${archiveName}`
+    const imagesFolder = `cypress/fixtures/image_issue_${issueId}`
+    const lastElement = images[images.length - 1]
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        for (let img of images) {
+            cy.imageGenerator(imagesFolder, img, width, height, color, posX, posY, labelName)
+        }
+        cy.wait(2000)
```

Comment:
Why should we wait here?

Reply:
Sometimes I encountered a situation when only 2 images were archived. The third image apparently did not exist at this point. So I decided to add a little waiting.

Reply:
Probably it is because the function ``imageGenerator`` works as async. Probably we can do this command returns a promise. Then ``cypress`` will wait till this command is done

https://docs.cypress.io/api/commands/task.html#Return-a-Promise-from-an-asynchronous-task

Reply:
I'll try to rework the function.  Thanks.